components/alert.css

```diff
@@ -0,0 +1,39 @@
+.alert-simple {
+   padding: 0.5rem;
+   border-style: solid;
+   border-width: 0.15rem;
```

Comment:
You can use shorthand property. For small values say 4px or smaller you can use px instead of rem